stores/sql/mysql/main.go

```diff
@@ -716,11 +716,11 @@ func (tx *MainDatabaseTx) ResetLostSectors(ctx context.Context, hk types.PublicK
 	return ssql.ResetLostSectors(ctx, tx, hk)
 }
 
-func (tx MainDatabaseTx) SaveAccounts(ctx context.Context, accounts []api.Account) error {
+func (tx MainDatabaseTx) SaveAccounts(ctx context.Context, owner string, accounts []api.Account) error {
 	// clean_shutdown = 1 after save
 	stmt, err := tx.Prepare(ctx, `
-		INSERT INTO ephemeral_accounts (created_at, account_id, clean_shutdown, host, balance, drift, requires_sync)
-		VAlUES (?, ?, 1, ?, ?, ?, ?)
+		INSERT INTO ephemeral_accounts (created_at, account_id, clean_shutdown, host, balance, drift, requires_sync, owner)
+		VAlUES (?, ?, 1, ?, ?, ?, ?, ?)
```

Comment:
The fact that we don't need to extend `ON DUPLICATE KEY UPDATE` caught my attention... should we alter the `UNIQUE` constraint here to take `OWNER` into account?

Reply:
The idea here was to draw attention to multiple owners using the same accounts which shouldn't happen under normal circumstances.

Reply:
Shouldn't we assert that though on database level and extend the unique constraint to cover both the account id and the owner field.

Reply:
Isn't that what we are doing? If you insert the same account for a different owner you will get a unique constraint violation the way it is now. If we expand the constraint over both id and owner, we allow multiple owners to reuse the same id.